Sources/TuistCore/Cache/CacheCategory.swift

```diff
@@ -11,6 +11,9 @@ public enum CacheCategory: String, CaseIterable, RawRepresentable {
 
     /// The manifests cache
     case manifests
+    
+    /// The edit projects cache
+    case generatedEditProjects
```

Comment:
Let's rename this one to `editProjects` to align with the name chosen for the directory.